documents/TestSuite/sxpbrlib/bsdf/addbsdf.mtlx

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<materialx version="1.36">
+  <nodegraph name="addbsdf_test" type="">
```

Comment:
Example using addbsdf to implement a normalized bsdf mix (a bit nonsensical since there already is a separate node for that, but the point here is to test the addbsdf node).
Note how the weight input on each bsdf node controls the blending amount for this lobe.